sdk/src/main/java/io/radar/sdk/RadarGoogleLocationClient.kt

```diff
@@ -47,11 +53,10 @@ internal class RadarGoogleLocationClient(
     ) {
         val priority = priorityForDesiredAccuracy(desiredAccuracy)
 
-        val locationRequest = LocationRequest().apply {
-            this.priority = priority
-            this.interval = interval * 1000L
-            this.fastestInterval = fastestInterval * 1000L
-        }
+        val locationRequest = LocationRequest.Builder(priority, interval * 1000L)
+            .setMaxUpdateAgeMillis(0)
```

Comment:
do we wanna do this for all `requestLocationUpdates`? Seems like a big change, maybe make it configurable and only for `trackVerified` to start?